### PR TITLE
Clear up confirmation

### PR DESCRIPTION
**Pull request by martypowell**
- Accounts for service changes by changing the address parts field
- Adds business name to confirmation page

@tmccoy529 @sgrg1 if you test this, you will need to get the latest from the service

### CODE REVIEW

File: src/components/formik/AddressLookupField.jsx [hunk 1]
```diff
@@ -47,14 +47,16 @@ const CustomInputComponent = ({
...
     } else {
       setFieldValue("businessAddress", value);
-      setFieldValue("businessAddressParts", {});
+      setFieldValue("address", {});
     }
   };
 
-  const handleAddressSelect = (value, item) => {
+  const handleAddressSelect = (value, item, test) => {
     setFieldValue("businessAddress", value);
```
Comment (tmccoy529) [inline src/components/formik/AddressLookupField.jsx]: what is "test" doing?

File: src/components/formik/AddressLookupField.jsx [hunk 2]
```diff
@@ -47,14 +47,16 @@ const CustomInputComponent = ({
...
-    setFieldValue("businessAddressParts", item);
+    setFieldValue("address", item);
     setWasSelected(true);
+
+    console.log(value, item, test);
   };
```
Comment (tmccoy529) [inline src/components/formik/AddressLookupField.jsx]: need to remove console.log

File: src/components/formik/AddressLookupField.jsx (hunk 1 above)
Comment (martypowell) [reply src/components/formik/AddressLookupField.jsx]: shoudl be gone, good catch.

File: src/components/formik/AddressLookupField.jsx (hunk 2 above)
Comment (martypowell) [reply src/components/formik/AddressLookupField.jsx]: yes

**Pull request merged**